pages/index.tsx

```diff
@@ -17,7 +24,7 @@ export default function Home() {
         </h1>
 
         <p className={styles.description}>
-          Get started by editing{' '}
+          Get started by editing{" "}
```

Comment:
what is this line here for ?

Reply:
automatically change due to mine vs code settings